proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/saml/HubResponseContainer.java

```diff
@@ -11,18 +11,21 @@
 
 public class HubResponseContainer {
 
-    private String pid;
-    private String eidasRequestId;
-    private URI destinationURL;
-    private URI issuer;
-    private Attributes attributes;
-    private VspScenario vspScenario;
-    private VspLevelOfAssurance levelOfAssurance;
-
-    public HubResponseContainer(HubResponseTranslatorRequest hubResponseTranslatorRequest, TranslatedHubResponse translatedHubResponse) {
+    private final String pid;
+    private final String eidasRequestId;
+    private final URI destinationURL;
+    private final URI issuer;
+    private final Attributes attributes;
+    private final VspScenario vspScenario;
+    private final VspLevelOfAssurance levelOfAssurance;
+
+    public HubResponseContainer(
+            final HubResponseTranslatorRequest hubResponseTranslatorRequest,
+            final TranslatedHubResponse translatedHubResponse,
+            final URI destination) {
         this.pid = translatedHubResponse.getPid().orElse(null);
         this.eidasRequestId = hubResponseTranslatorRequest.getEidasRequestId();
-        this.destinationURL = hubResponseTranslatorRequest.getDestinationUrl();
```

Comment:
`hubResponseTranslatorRequest.getDestinationUrl()` is fine - the `destinationUrl` is contained in `hubResponseTranslatorRequest` and should not be a separate argument of the constructor - this is confusing